servers.json

```diff
@@ -955,5 +955,13 @@
         "addresses": [
             "teamfight.cc"
         ]
+    },
+    {
+        "id": "CobelPvP",
```

Comment:
This must match the regex specified for id fields; can only consist of lowercase letters and whole numbers